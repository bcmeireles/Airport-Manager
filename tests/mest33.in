a LIS Portugal Lisboa
a POR Portugal Lisboa
v TT6 LIS POR 14-02-2022 14:00 2:00 200
r TT6 14-02-2022 ZZZZZZZZZZZZZZ 0
r TT6 14-02-2022 ZZZZZZZZZZZZZZ 1
t 20-02-2022
r TT7 01-01-2021 0000000 205
r TT7 01-01-2021 aaaaaaa 205
r TT7 01-01-2021 zzzzzzzzzzzzzz 205
r TT7 01-01-2021 00000000000000 205
r TT6 14-02-2022 ZZZZZZZZZZZZZZ 1
r TT6 14-02-2022 ZZZZZZZZZZZZZZA 205
r TT6 14-02-2022 ZZZZZZZZZZZZZZA 0
r XXX 14-02-2022
r TT6 14-02-2022
q