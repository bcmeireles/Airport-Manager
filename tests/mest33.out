airport LIS
airport POR
invalid passenger number
20-02-2022
invalid reservation code
invalid reservation code
invalid reservation code
TT7: flight does not exist
ZZZZZZZZZZZZZZ: flight reservation already used
too many reservations
invalid date
XXX: flight does not exist
invalid date
